cmd/restore/main.go

```diff
@@ -29,6 +31,7 @@ func init() {
 	schemaName = flag.String("schema", "public", "Schema name")
 	targetTable = flag.String("target-table", "", "Target table name (optional)")
 	backupDir = flag.String("backup-dir", "", "Backups dir")
+	debug = flag.Bool("debug", false, "Toggle debug mode")
```

Comment:
Ideally there should be loglevel instead of just debug

Reply:
Those were added in the latest commit.